packaging/nnstreamer.spec

```diff
@@ -550,6 +558,7 @@ cp -r result %{buildroot}%{_datadir}/nnstreamer/unittest/
 %defattr(-,root,root,-)
 %license LICENSE
 %{_prefix}/lib/nnstreamer/decoders/libnnstreamer_decoder_*.so
+%{_prefix}/lib/nnstreamer/converters/libnnstreamer_converter_*.so
```

Comment:
Please investigate Tizen platform images. (look at various .packages files of Tizen:Unified image repositories at download.tizen.org)
If there are images without flatbuf, converter_flabuf should be subpackaged.

Reply:
I checked http://download.tizen.org/snapshots/tizen/unified/, armv7l, aarch64, i686 and x86_64 have flatbuf rpm packages.